test/test_garcia.jl

```diff
@@ -28,9 +29,12 @@ Y = Dataset(x)
     max_1_idx = Peaks.maxima(N,min_dist)
     max_2_idx = Peaks.maxima(N2,min_dist)
 
-    @test max_1_idx == max_2_idx
```

Comment:
I had to adjust it, because I changed the time series length to the values described in the paper. - Just to see, that it still does not match the results proposed there.